test/unit/lib/errors.test.js

```diff
@@ -26,8 +26,7 @@
 
```

Comment:
Please delete this test as it is no longer applicable.

Reply:
Deleted